Changes

```diff
@@ -69,6 +69,10 @@ Improvements
 - MeshTangents : Added support for additional computation modes (#3030).
 - GafferImageUI: Added DWA compression presets (#3153).
 - Numeric Bookmarks : Added serialisation to preserve numeric bookmarks across sessions (#3157).
+- SetExpressions : Support wildcard characters in set names (#3172).
```

Comment:
Avoid the weird imperative mood that we use for commit messages - our main audience is the end user here, not  developers.